app/styles/partials/footer.scss

```diff
@@ -33,3 +33,68 @@ footer {
     right: 0 !important;
   }
 }
+
+
+@media screen and (min-width : 1200px) {
+    #main-container {
```

Comment:
@sumedh123 we need these styles applied **only** for pages with a side rail. We don't want this on all pages.